lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,17 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $user = user_load($data->uid);
+    $this->user = [
+      'drupal_id' => $data->uid,
+      'northstar_id' => dosomething_helpers_isset($user->uuid, 'id'),
+      'first_name' => dosomething_helpers_isset($user->field_first_name[LANGUAGE_NONE][0]['value'], 'first_name'),
+      'last_initial' => dosomething_helpers_isset($user->field_last_name[LANGUAGE_NONE][0]['value'], 'last_initial'),
+      'photo' => dosomething_helpers_isset($user->photo, 'photo'),
+      'country' => dosomething_helpers_isset($user->field_address[LANGUAGE_NONE][0]['country'], 'country'),
```

Comment:
you should be able to pass the field first through `dosomething_helpers_extract_field_data()` which deals with language stuff w/out hardcoding it, and then run the `isset(...)` around it :)